### PR TITLE
Release/1.5.0

### PR DESCRIPTION
**Pull request by cqc-melf**

### CODE REVIEW

**Comment by cqc-melf** [comment]
This PR updates the requirements to the new released version from today.

**Comment by cqc-melf** [comment]
I have run the checks locally. Should all work now.

File: examples/python/entanglement_swapping.py [hunk 1]
```diff
@@ -237,7 +237,8 @@ def make_noise_model(dep_err_rate, ro_err_rate, qubits):
     )
     # Add depolarising error to CX gates between any qubits (implying full connectivity)
     for i, j in product(qubits, repeat=2):
-        model.add_quantum_error(dep_err, ["cx"], [i, j])
+        if(i != j):
```
Comment (cqc-alec) [inline examples/python/entanglement_swapping.py:240]: Should maybe add a black formatting check to the CI.

**Review by cqc-alec — APPROVED** [review]

File: examples/python/entanglement_swapping.py (hunk 1 above)
Comment (cqc-melf) [reply examples/python/entanglement_swapping.py:240]: Yes we should do that, will make a task

**Pull request merged**